### PR TITLE
required fix for max validators count

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

**Review by IlyaVi — APPROVED** [review]

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts [hunk 1]
```diff
@@ -209,20 +209,20 @@ class OperatorStore extends BaseStore {
...
-    return this.operatorValidatorsLimit;
   }
 
   /**
    * Check if operator registrable
    */
-  isOperatorRegistrable(validatorsRegisteredCount: number) {
-    return this.operatorValidatorsLimit > validatorsRegisteredCount;
+  isOperatorRegistrable(validatorsRegisteredCount: number): boolean {
+    const result = Math.min(validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE, this.operatorValidatorsLimit);
```
Comment (alexstolr) [inline src/app/common/stores/applications/SsvWeb/Operator.store.ts:224]: Something here isn't clear to me.
there are 2 scenarios:
1. result = validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE
2.  result = this.operatorValidatorsLimit

in first case:
this.operatorValidatorsLimit > result will result in true.
in second case:
this.operatorValidatorsLimit > this.operatorValidatorsLimit is always false.

it seems that this is equivalent to just returning:
 this.operatorValidatorsLimit > validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE

Or you possibly meant to check something else?

**Review by alexstolr — CHANGES_REQUESTED** [review]

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts:224]: u right, it's equivalent to this.operatorValidatorsLimit > validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE

**Review by IlyaVi — APPROVED** [review]

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts [hunk 2]
```diff
@@ -209,20 +209,20 @@ class OperatorStore extends BaseStore {
...
     }
-    return this.operatorValidatorsLimit;
   }
 
   /**
    * Check if operator registrable
    */
-  isOperatorRegistrable(validatorsRegisteredCount: number) {
-    return this.operatorValidatorsLimit > validatorsRegisteredCount;
+  isOperatorRegistrable(validatorsRegisteredCount: number): boolean {
```
Comment (meshin-dev) [inline src/app/common/stores/applications/SsvWeb/Operator.store.ts:223]: typo, should be `isOperatorRegisterable`. But better name in common would be `operatorReachedValidatorsLimit`

**Review by meshin-dev — CHANGES_REQUESTED** [review]

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts]: @alexstolr @meshin-dev done

File: src/app/common/stores/applications/SsvWeb/Operator.store.ts (hunk 2 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Operator.store.ts]: renamed

**Review by alexstolr — APPROVED** [review]

**Review by meshin-dev — APPROVED** [review]

**Pull request merged**